cocos/core/scene-graph/base-node.ts

```diff
@@ -332,6 +332,8 @@ export class BaseNode extends CCObject implements ISchedulable {
     // record scene's id when set this node as persist node
     public _originalSceneId = '';
 
+    private _childrenChanged = 0;
```

Comment:
`_childTreeVersion`

1. It means the whole tree update, so children could cause miss understanding.
2. Changed is often considered a flag or a status, so I think version could be better